litellm/llms/vertex_ai.py

```diff
@@ -299,10 +299,12 @@ def completion(
         print_verbose(
             f"VERTEX AI: vertex_project={vertex_project}; vertex_location={vertex_location}"
         )
-        creds, _ = google.auth.default(quota_project_id=vertex_project)
-        print_verbose(
-            f"VERTEX AI: creds={creds}; google application credentials: {os.getenv('GOOGLE_APPLICATION_CREDENTIALS')}"
-        )
+
+        creds, _ = google.auth.default(quota_project_id=vertex_project) if optional_params.get("credentials") is None else (optional_params.get("credentials"), None)
+        if optional_params.get("credentials") is None:
```

Comment:
why print verbose only on None?

Reply:
Refactoring principle of not touching anything that's currently working/avoiding any api behavior change at all. 

@krrishdholakia Want me to remove this?